.github/workflows/model-analysis-weekly.yml

```diff
@@ -14,6 +14,7 @@ jobs:
   model-analysis:
     needs: docker-build
     runs-on: runner
+    timeout-minutes: 10080 # Set job execution time to 7 days(default: 6 hours)
```

Comment:
After 24h GH auto token will expire, so if you want to do even longer jobs we must make a PAT token for it
https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#jobsjob_idtimeout-minutes